docs/en/query_language/create.md

```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
 
 Supported compression algorithms:
 
-- `NONE` - no compression for data applied
+- `NONE` — no compression for data applied.
 - `LZ4`
-- `LZ4HC(level)` - (level) - LZ4\_HC compression algorithm with defined level.
+- `LZ4HC(level)` — (level) - LZ4\_HC compression algorithm with defined level.
 Possible `level` range: \[3, 12\]. Default value: 9. Greater values stands for better compression and higher CPU usage. Recommended value range: [4,9].
-- `ZSTD(level)` - ZSTD compression algorithm with defined `level`. Possible `level` value range: \[1, 22\]. Default value: 1.
+- `ZSTD(level)` — ZSTD compression algorithm with defined `level`. Possible `level` value range: \[1, 22\]. Default value: 1.
 Greater values stands for better compression and higher CPU usage.
-- `Delta(delta_bytes)` - compression approach when raw values are replace with difference of two neighbour values. Up to `delta_bytes` are used for storing delta value.
+- `Delta(delta_bytes)` — compression approach when raw values are replace with difference of two neighbour values. Up to `delta_bytes` are used for storing delta value.
```

Comment:
as far as I understand delta_bytes is not only the maximum size of delta, but also a byte size of raw values to subtract (which is important to understand to use this properly)